tests/board.tests.cpp

```diff
@@ -0,0 +1,25 @@
+#include <gtest/gtest.h>
+
+#include "Battlemines_2/board.h"
+
+TEST(Placement, board_is_created_properly){
```

Comment:
#nice, but #should not create your own tests - Franco is the one who will do this instead. You should be writing unit tests for Agus' code!